projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/credentials/KerberosCredentialsRepository.java

```diff
@@ -62,7 +62,7 @@ private void checkDependencies() {
             "Missing required dependencies. "
                 + " kadmin are missing or are not found on PATH."
                 + "Possible reason: "
```

Comment:
```
if (result == null || result.getExitValue() != 0) {
        log.error(
             "Missing required dependencies. "
                 + " kadmin are missing or are not found on PATH."
                 + "Possible reason: "
                 + result.getOutput().getUTF8()
                 + (result != null ? result.getOutput().getUTF8() : "")
```
So in case of result == null we print an empty-reasoned error message?

Reply:
yes, as we don't have a valid reason.